plugins/wazo-snom/common/common.py

```diff
@@ -156,6 +157,14 @@ def __init__(self, app, plugin_dir, gen_cfg, spec_cfg):
 
     http_dev_info_extractor = BaseSnomHTTPDeviceInfoExtractor()
 
+    def _add_uxm_firmware(self, raw_config):
+        f = glob.glob(self._tftpboot_dir + u'/firmware/snomD7C-*.bin') + glob.glob(self._tftpboot_dir + u'/firmware/snomUXM-*.bin')
+        if len(f):
+            if re.match(r"^.*\/snomUXM-.*.bin$", max(f, key=os.path.getmtime)):
+                raw_config[u'XX_uxm_firmware'] = 'uxm'
+            if re.match(r"^.*\/snomD7C-.*.bin$", max(f, key=os.path.getmtime)):
```

Comment:
I think that having the `glob` in two different lists would avoid the need to do a regex

Reply:
Unfortunately no, as sorting by modification date (to get the most recent downloaded firmware) would then not be possible.